Polyhedron/demo/Polyhedron/Scene_polyhedron_item.h

```diff
@@ -24,6 +24,7 @@ class SCENE_POLYHEDRON_ITEM_EXPORT Scene_polyhedron_item
         : public CGAL::Three::Scene_item{
     Q_OBJECT
 public:
+  typedef Polyhedron FaceGraph;
     enum STATS {
       NB_VERTICES = 0,
```

Comment:
- In my opinion the naming convention should be the one of CGAL for types: `Face_graph`.
- What is more: should not you also add a function `Face_graph* face_graph()`?